docs/tutorial.rst

```diff
@@ -875,62 +876,54 @@ Again, methods to do this are discussed in the
 Incorporating selection
 =======================
 
-Here are some examples of how to incorporate selection.
-This API is **not final**, and will probably change!
+There two general ways to incorporate selection into a simulation.
```

Comment:
```suggestion
There are two general ways to incorporate selection into a simulation.
```

Reply:
so i dont get from this text what are the two general ways. But maybe I just need to go over everything.

Reply:
The two ways are selection on intervals and selection on a single mutation site. I agree we should spell this out more clearly, also maybe linking these things to actual processes people will use them for, namely background selection and sweeps.